src/ahbicht/content_evaluation/content_evaluation_result.py

```diff
@@ -66,3 +66,60 @@ def deserialize(self, data, **kwargs) -> ContentEvaluationResult:
                         result.requirement_constraints[rc_key] = ConditionFulfilledValue(enum_value.value)
                         break
         return result
+
+
+@attr.s(auto_attribs=True)
+class ContentEvaluationPrerequisites:
+    """
+    For expressions (that do not contain any unresolved package) it's possible to pre-generate all possible outcomes of
+    a content evaluation. ContentEvaluationPrerequisites are the answer to the question:
+    'Which information do I need to provide in a ContentEvaluationResult in order to evaluate a given expression?'
```

Comment:
Ich weiß Prerequisites sind dafür vermutlich ein guter Name, aber der Vollständigkeit halber möchte ich festhalten, das edi@energy ab FV2022 die Requirment Constraints "Voraussetzungen" nennt.

Reply:
hast du denn einen besseren? Ich dachte alternativ an ContentEvaluationResultTemplate

Reply:
Naming ist mal wieder hard...
Was mich so ein bisschen stört, ist dass du es gerade danach benennst, was du damit vorhast und nicht was es ist.
Was in die Richtung SortedConditionSet(Of/For)Expression? Also natürlich ohne die Sonderzeichen sondern eine Version davon^^

Reply:
Ah, mit ein paar Stunden Abstand verstehe ich was du mit "sorted" meinst. Es geht nicht darum, dass 2 vor 3 kommt sonder um Sortenreinheit.
Nächster Vorschlag: HomogeneousKeyExtract?
Du hast schon recht mit dem "was es ist" vs. "wofür ich es nutzen will". Das wort "extract" träfe vllt nicht 100% genau was es ist aber wie es entsteht.

Reply:
CategorizedKeyExtract?

Reply:
https://github.com/Hochfrequenz/ahbicht/pull/91/commits/1804498193491a73690fe6cf2a7f74c11d46f88d